Makefile

```diff
@@ -21,14 +21,10 @@ download_prebuilt:
 .PHONY: install
 install: printer.arm
 	ssh-add
-	ssh root@$(host) systemctl stop printer
+	ssh root@$(host) systemctl stop printer 2>&1 >/dev/null
```

Comment:
Let's just check if the service is running first.

    if ssh root@$(host) systemctl is-active --quiet printer;
    then
        [stop service]
    fi

Reply:
```suggestion
	if ssh root@$(host) systemctl is-active --quiet printer;
	then
	    [stop service]
	fi
```